scripts/ps-modules/Build/Build.psm1

```diff
@@ -258,6 +295,11 @@ function Run-InstallPackageStep
    )
    Write-Banner -Level 3 -Title "Install package step: $packageAndFeatures"
    $triplets = (Get-Triplets -linkType $linkType -buildType $buildType -customTriplet $customTriplet)
+
+   if( Check-RequiresEmscripten -triplets $triplets ) {
+      Install-Emscripten -version "3.1.58"
```

Comment:
Nit: Magic string.  Probably fine for now, since we just have one.  I am curious if this version is something we'll want to control globally for all WASM builds, or on a package-by-package basis.  Either way, we could look at adding this to preconfigured-packages.json down the road, either as a per-package optional property and/or as a global setting in a new "tools" section, or something like that.  Maybe we can revisit it if we end up moving the pango WASM build over here?

Reply:
Ok, I at least moved it into a variable that we could easily then pass in from the caller if needed.